packages/vats/src/lib-chainStorage.js

```diff
@@ -113,3 +113,46 @@ export async function makeStorageNode(chainStorage, childName) {
   return storageNode;
 }
 harden(makeStorageNode);
+
+/**
+ * For testing, creates a chainStorage root node over an in-memory map
+ * and exposes both the map and the sequence of received messages.
+ *
+ * @param {string} rootPath
+ */
+export const makeFakeStorageKit = rootPath => {
+  /** @type {Map<string, unknown>} */
```

Comment:
Why `unknown`?
```suggestion
  /** @type {Map<string, string>} */
```

Reply:
from before I new that it was always `string` :) 

updated type and instances to match the Golang